static/js/stats/stats.js

```diff
@@ -725,6 +725,224 @@ function populate_number_of_users(data) {
     $('#all_time_actives_button').addClass("selected");
 }
 
+function populate_messages_read_over_time(data) {
```

Comment:
Not displaying read_interaction. I wanted some feedback on how we want the read interaction to be displayed. Like we should have a button to switch between total read counts and read interaction or we want a separate graph for that with more information.
Also where we need to place this graph is also a matter to discuss.

Reply:
Yeah, I'm not sure either.  I think there's a good chance that after some testing, we'll decide we only want one or the other; it just seems like we should be doing the data collection for both for now so that we can play with the options on chat.zulip.org after we have a couple weeks of real data.